### DISCUSSION TITLE
Learning about perceptual hash

### DISCUSSION THREAD

**Discussion by dotronghieu** [coenm/ImageHash#35] [Q&A]
Hi. I'm last year of college student, my project right now is make a website for selling and storing images, I start to lear about perceptual hash but still not very well understand about it. I know about the Hamming distance but I didn't understand the way you compare the hash. And if I want to store the Ulong Hash to SQL DB, what type should I use?  If you have time, mailed me at hieudtse130087@fpt.edu.vn, I will very thankful

**Comment by coenm** [comment]
I suggest you read the following articles to learn more about the perceptual hashing algorithms used in this library.

http://www.hackerfactor.com/blog/index.php?/archives/432-Looks-Like-It.html
http://www.hackerfactor.com/blog/index.php?/archives/529-Kind-of-Like-That.html
